README.md

```diff
@@ -627,6 +629,27 @@ Examples:
 - `ref+gitlab://gitlab.com/11111/password`
 - `ref+gitlab://my-gitlab.org/11111/password?ssl_verify=true&scheme=https`
 
+### 1Password Connect
+
+For this provider to work you require a working and accessible [1Password connect server](https://developer.1password.com/docs/connect).
+The following env vars have to be configured:
+- `OP_CONNECT_HOST`
```

Comment:
what's the meaning of OP? maybe it's better to use the full name.

Reply:
These are the default env variables the 1Password connect SDK uses (I guess it's an abbreviation for for OnePassword?)
But if the naming is an issue I'm happy to define custom environment variables (e.g. `ONEPASSWORDCONNECT_HOST` and `ONEPASSWORDCONNECT_TOKEN`) and use them to create the 1password connect API client.

Reply:
got it. no need to change.